File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dicc={}
        lista=[]
        frequent_num=[]
        for num in nums:
            if num not in dicc:
                dicc[num]=1
            else:
                dicc[num]+=1
        for num in dicc:
            lista.append((dicc[num],num))
        lista=sorted(lista,reverse=True)
        for x in range(k):
            frequent_num.append(lista[x][1])
        return frequent_num

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int temp=0,count=0;

        for(int x:nums)
        {
            if(x==1)
            {
                count++;
                if (count>temp)
                {
                    temp=count;
                }
            }
            else
            {
                count=0;
            }
        }
        return temp;
    }
}